### PR TITLE
Install atomic-openshift-utils instead of openshift-ansible-playbooks

### PR DESCRIPTION
**Pull request by jhadvig**
... and remove unnecessary steps
@stevekuznetsov PTAL

### CODE REVIEW

File: config/test_cases/test_branch_origin_extended_conformance_install_update.yml [hunk 1]
```diff
@@ -77,11 +77,6 @@ extensions:
       script: |-
         sudo yum install -y atomic-openshift-utils
     - type: "script"
-      title: "check out the correct branch of aos-cd-jobs"
-      repository: "aos-cd-jobs"
-      script: |-
-        git checkout sjb
```
Comment (stevekuznetsov) [inline config/test_cases/test_branch_origin_extended_conformance_install_update.yml]: Isn't the inventory dir only in this branch?

Comment (jhadvig) [inline config/test_cases/test_branch_origin_extended_conformance_install_update.yml]: it is, but we already checkout the `sjb` branch before -> https://github.com/jhadvig/aos-cd-jobs/blob/cleanup/config/test_cases/test_branch_origin_extended_conformance_install_update.yml#L14

**Comment by stevekuznetsov** [comment]
Can you push up the generated XML for only the `test_branch_origin_extended_conformance_install` job (so we don't break PR test or merge) and run it once just to make sure this does exactly what you need?

**Comment by jhadvig** [comment]
Updated job running  https://ci.openshift.redhat.com/jenkins/job/test_branch_origin_extended_conformance_install_update/21/

**Comment by jhadvig** [comment]
@stevekuznetsov I've re-generated the job with the `oreg_url` that you suggested to try... the job is running at  
https://ci.openshift.redhat.com/jenkins/job/test_branch_origin_extended_conformance_install_update/22/console

**Comment by stevekuznetsov** [comment]
cool cool cool

**Comment by jhadvig** [comment]
Closing in favor of https://github.com/openshift/aos-cd-jobs/pull/131

**Pull request closed without merging**